java/maven.embedder/build.xml

```diff
@@ -21,16 +21,13 @@
 -->
 <project basedir="." default="build" name="java/maven.embedder">
     <import file="../../nbbuild/templates/projectized.xml"/>
-    <target name="-release.files" depends="projectized.-release.files">
+    <target name="-release.files" depends="projectized.-release.files,check-binariesembedded-list">
```

Comment:
the task is now part of the build

Reply:
+1 but maybe we should add the download check, if artifcat coordinate is semanticaly correct but not present on central would it fail ?

Reply:
I tested the list manually with this: https://github.com/apache/netbeans/pull/5939#issuecomment-1545628713

we should think about the whole approach a bit more since this doesn't feel right to me. Instead of validating the list after doing an error prone manual update we should think about generating it from https://github.com/apache/maven/blob/master/pom.xml or something equivalent.